instagram/src/postcontainer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+//import Comments from './comments.js';
+
+export const Postcontainer = (props) => {
+    return (
+        <div className= 'Post-Wrapper'>
+            {props.dummyData.map(post => {
+                return (
+                    <div key= {post.state}>
```

Comment:
On this line - {post.state} is not referencing anything because the callback parameter of "post" doesn't have any state.